docs/v0.6/en/user-guide/operations/configuration.md

```diff
@@ -193,15 +212,15 @@ All of these protocols except HTTP and gRPC are optional, the default values are
 |            | addr         | String  | Server address, "127.0.0.1:4003" by default                                     |
 |            | runtime_size | Integer | The number of server worker threads, 2 by default                               |
 
-#### Node options
+<!-- #### Node options
```

Comment:
We still need this section,  because there is a `node_id` for datanode.

Reply:
`node_id` option is in the `Datanode-only configuration` section.